pad.js

```diff
@@ -0,0 +1,13 @@
+pry = require('pryjs')
+
+var pad = Array.prototype.pad = function(min, pad) {
```

Comment:
we don't want to add things to the Array prototype, we just want to create stand-alone functions that handle a specific piece of logic